modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -43,59 +49,123 @@ sourceFiles:
       storageClassName: image-registry-sc
       volumeMode: Filesystem
   # persistent volume
-  - fileName: ImageRegistryPV.yaml <3>
+  - fileName: ImageRegistryPV.yaml <2>
     policyName: "pv-for-image-registry"
     metadata:
       annotations:
-        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+        ran.openshift.io/ztp-deploy-wave: "100"
   # image registry config
-  - fileName: ImageRegistryConfig.yaml <5>
+  - fileName: ImageRegistryConfig.yaml <3>
     policyName: "config-for-image-registry"
     complianceType: musthave <5>
     metadata:
       annotations:
-        ran.openshift.io/ztp-deploy-wave: "100" <6>
+        ran.openshift.io/ztp-deploy-wave: "100"
     spec:
       storage:
         pvc:
           claim: "image-registry-pvc"
 ----
-<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is appropriate for an individual site.  ZTP deploy waves are used to order how policies are applied to the spoke cluster. All policies created by `PolicyGen` have a ztp deploy wave by default.
-<2> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
-<4> Set the appropriate value for `ztp-deploy-wave` as in note 1.
-<5> Configure registry to point to the PVC created above.
-<6> Set the appropriate value for `ztp-deploy-wave` as in note 1.
+<1> Set the appropriate value for `ztp-deploy-wave` depending on whether you are configuring image registries at the site, common, or group level. `ztp-deploy-wave: "100"` is suitable for development or testing because it allows you to group the referenced source files together.
+<2> In `ImageRegistryConfig.yaml`, ensure that the `spec.local.path` field is set to `/var/imageregistry` to match the value set for the `mount_point` field in the `SiteConfig` CR.
+<3> Do not set `complianceType: mustonlyhave`. `mustonlyhave` causes the registry pod deployment to fail.
```

Comment:
Could double check the info here ? https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/gitops-subscriptions/argocd/ImageRegistry.md#installation

e.g <2> should be talking about ImageRegistryPV -- here `/var/imageregistry` is by default. https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/ImageRegistryPV.yaml#L16 

user can change it if they want but should be matched with the disk partition/SiteConfig.

I think that's why SiteConfig's disk partition config should be specified here as well? 

A note: the siteconfig disk partition feature, although super useful in general purpose partitioning, it has only been tested in ImageRegistry context. E.g The disk partitioning is done with a MachineConfig CR which is literally called "image-registry-partition-mc" . Not sure if it's ready to go under "advanced feature" yet? 

https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/extra-manifest/image-registry-partition-mc.yaml.tmpl

Reply:
OK - I updated the error at `<2>`. Also moved the configuring disk partitioning procedure back where it was. PTAL: https://53498--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.html#ztp-add-local-reg-for-sno-duprofile_ztp-advanced-policy-config